ci/deps/travis-37-cov.yaml

```diff
@@ -31,11 +29,13 @@ dependencies:
   - odfpy
   - openpyxl
   - pandas-gbq
+  - google-cloud-bigquery>=1.27.2 # GH 36436
```

Comment:
This has the side effect of setting `google-cloud-core>=1.4.1`, which sounds like is the true workaround for #36436

Reply:
Looking at a recent CI failure https://travis-ci.org/github/pandas-dev/pandas/jobs/728321735
```
google-cloud-bigquery     1.25.0             pyh32f6830_0    conda-forge

google-cloud-bigquery-core 1.25.0           py37hc8dfbb8_0    conda-forge

google-cloud-bigquery-storage 0.8.0                      py_0    conda-forge

google-cloud-bigquery-storage-core 0.8.0                    py37_0    conda-forge

google-cloud-core         1.4.1                      py_0  
```

The google-cloud-core version was already 1.4.1. But the bug was still present. I wonder where did they actually fix it.